src/openrct2/PlatformEnvironment.cpp

```diff
@@ -18,6 +18,8 @@
 
 using namespace OpenRCT2;
 
+using DIRBASE_VALUES = u8string[DIRBASE_COUNT];
```

Comment:
This seems to be already defined in PlatformEnvironment.h, ideally this is also std::array so we can pass by const reference rather than copy.

Reply:
Also see PlatformEnvironment.cpp line 83, it should probably use the same type.